package.json

```diff
@@ -34,11 +34,11 @@
     "commands": [
       {
         "command": "extension.gotoOnlineLink",
-        "title": "Goto Online Link"
+        "title": "Gitlink: Goto Online Link"
```

Comment:
Would you please update to `GitLink`? Thanks~

Reply:
sure